release-notes/5.0/api-diff/netcoreapp3.1/.Net/5.0_System.Runtime.Intrinsics.md

```diff
@@ -0,0 +1,35 @@
+# System.Runtime.Intrinsics
```

Comment:
General intrinsics LGTM. CC @CarolEidt, @echesakovMSFT, @kunalspathak 

Also CC. @pgovind for the Numerics interop APIs